carapace-server/src/main/java/org/carapaceproxy/server/config/NetworkListenerConfiguration.java

```diff
@@ -19,83 +19,85 @@
  */
 package org.carapaceproxy.server.config;
 
-import java.util.Collections;
-import java.util.Set;
-
 import io.netty.channel.group.ChannelGroup;
 import io.netty.channel.group.DefaultChannelGroup;
 import io.netty.util.concurrent.DefaultEventExecutor;
-import lombok.Data;
+import java.util.Set;
+import lombok.Getter;
+import org.carapaceproxy.core.ForwardedStrategies;
 
 /**
  * Listens for connections on the network
  */
-@Data
+@Getter
 public class NetworkListenerConfiguration {
 
     public static final Set<String> DEFAULT_SSL_PROTOCOLS = Set.of("TLSv1.2", "TLSv1.3");
+    public static final int DEFAULT_SO_BACKLOG = 128;
+    public static final int DEFAULT_KEEP_ALIVE_IDLE = 300;
+    public static final int DEFAULT_KEEP_ALIVE_INTERVAL = 60;
+    public static final int DEFAULT_KEEP_ALIVE_COUNT = 8;
+    public static final int DEFAULT_MAX_KEEP_ALIVE_REQUESTS = 1000;
+    public static final String DEFAULT_FORWARDED_STRATEGY = ForwardedStrategies.preserve().name();
```

Comment:
In the PR description, you wrote that the default is REWRITE, but here you use PRESERVE

Reply:
It is preferable to have REWRITE as the default.

Reply:
> In the PR description, you wrote that the default is REWRITE, but here you use PRESERVE

Oh, you're right, sorry!

> It is preferable to have REWRITE as the default.

OK

Reply:
Make sense..ok to set default to preserve